product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -216,9 +223,22 @@ Generating report ....
 Report generation complete.
 ```
 
+!!!  Note 
```

Comment:
Is this note supposed to be rendered in a Note box like other notes?  If so, it seems like that is not happening in the latest staging deployment.